### PR TITLE
Frontend Paddle :  repeat_interleave

### LINKED ISSUE #23081: repeat_interleave

### PR DESCRIPTION
**Pull request by dhanush-2501**
# PR Description 
Added frontend and test for paddle repeat_interleave

## Related Issue 

Close #23081

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### Socials: 
 https://twitter.com/coder_dhanush

### CODE REVIEW

**Comment by lucasalavapena** [comment]
Thanks for making a clean PR :smile: !

**Comment by dhanush-2501** [comment]
@lucasalavapena is it good to merge now?

**Comment by lucasalavapena** [comment]
> @lucasalavapena is it good to merge now?

Sorry, I just published my review comment, I was double checking this with other team members

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 1]
```diff
@@ -445,6 +448,40 @@ def test_paddle_gather(
...
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+    backend_fw,
+):
+    dtype, values, repeats, axis, _ = dtype_values_repeats_axis_output_size
+
+    helpers.test_frontend_function(
+        input_dtypes=dtype[1],
```
[the remark this answers is not in the document]
Comment (dhanush-2501) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:473]: When i used dtype[0] only Tensorflow is passing. All the other shows float casting error. Can you help me to include both types? How can I do that?

[the remark this answers is not in the document]
Comment (lucasalavapena) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:473]: See my new comment :smile:

**Comment by dhanush-2501** [comment]
@lucasalavapena  Fixed it. have a look on it now.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhanush-2501) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py]: @lucasalavapena Yes it worked. thanks for the help mate.

**Comment by lucasalavapena** [comment]
@dhanush-2501 Looks great. LGTM!

**Pull request merged**